File: flybanner/src/main/java/com/xia/flybanner/holder/FBViewHolderCreator.java
package com.xia.flybanner.holder;

import android.view.View;

/**
 * @author weixia
 * @date 2019/4/16.
 */
public interface FBViewHolderCreator {

    FBHolder createHolder(View itemView);

    int getLayoutId();
}

File: flybanner/src/main/java/com/xia/flybanner/holder/FBViewHolderCreator.kt
package com.xia.flybanner.holder

import android.view.View

/**
 * @author weixia
 * @date 2019/4/16.
 */
interface FBViewHolderCreator {

    fun getLayoutId(): Int

    fun createHolder(itemView: View): FBHolder<*>
}
